### ISSUE TITLE
Build failure on windows with llvm 11.0

### ISSUE THREAD

**Issue by bin** [MaskRay/ccls#720]
I am attempting to build ccls on windows 10 with the latest release 0.20201025, Clang and LLVM 11.0, and Visual Studio 2019.  When compiling, I receive the following error:

```
ccls\src\messages\workspace.cc(41,79): error C2679: binary '=': no operator found which takes a right-hand operand of type 'llvm::StringRef' (or there is no acceptable
conversion)
```

Let me know if I can send any more information to help.

**Comment by jamiltron** [comment]
I am also receiving this error, unfortunately. Similar setup.

**Comment by MaskRay** [comment]
```bash
cmake -GNinja -H. -BBundled11 -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=$HOME/llvm-prebuilt/clang+llvm-11.0.0-x86_64-linux-gnu-ubuntu-20.04
ninja -C Bundled11
```

It works on Linux. I suspect `StringRef` does not implicitly convert to `std::string_view` on MSVC.

**Comment by anurag-23** [comment]
Please refer to https://github.com/MaskRay/ccls/pull/747#issuecomment-762045660

This is an issue with MSVC incorrectly reporting the value of __cplusplus

**Comment by MaskRay** [comment]
Fixed by #747 . Thanks everyone participating in the discussion:)
